src/main/java/com/jcraft/jsch/ChannelSftp.java

```diff
@@ -643,6 +653,32 @@ public void _put(InputStream src, String dst, SftpProgressMonitor monitor, int m
         throw (SftpException) e;
       throw new SftpException(SSH_FX_FAILURE, e.toString(), e);
     }
+  
+  public boolean checkChunkFailed(InputStream src, String dst, String ChunkStart)throws SftpException{
```

Comment:
1. The argument `InputStream src` appears to be unused in this method, so why is it needed?
2. The argument `String ChunkStart` should be changed to not start with a capital letter.
-- Perhaps something like `String chunkStart` instead?

Reply:
1. Removed the argument InputStream src from the method.
2. Changed the parameter name to startChunk